project3/index.html

```diff
@@ -0,0 +1,246 @@
+</div>
+ 
+<div class="content">
+    <form action="">
+   
+    <fieldset>
+    <legend> IDENTITY</legend>
+    <label for="real-name">Real Name</label>
+    <input id="real-name" type="text" placeholder="" required>
+
+    <label for="alias"> Alias </label>
+    <input id="alias" type="text"placeholder="" required>
+    
+    <label for="email">Email</label>       
+    <input id="email" type="email" placeholder=""required>
+    
+    </fieldset>
+    
+
+    <fieldset>
+    <legend>PERSONAL CHARACTERISTICS</legend>
+
+    <div class="age">
+    <label for="age">Age</label>       
+    <input id="age" type="number" placeholder=""required>
+    </div>
+    </fieldset>
+
+    <fieldset class="half">
+      <legend class="category">Category</legend>
+     <p>Please select as many as apply to your origin and how you gained your powers.</p>
+            
+            
+            <label for="alien">
+            <input id="alien"type="checkbox">
+            Alien
+          </label>
+          <label for="cyborg">
+            <input id="cyborg"type="checkbox">
+            Cyborg
+          </label>
+            <label for="experiment">
+            <input id="experiment"type="checkbox">
+            Experiment
+          </label>
+          <label for="human">
+            <input id="human"type="checkbox">
+            Human
+          </label>
+          <label for="mutant">
+            <input id="mutant"type="checkbox">
+            Mutant
+          </label>
+          <label for="inhuman">
+            <input id="inhuman"type="checkbox">
+            Inhuman</label> 
+    </fieldset>  
+          
+  <fieldset class="half">
+  <legend class="gender">Gender</legend>
+            <label for="female">
+            <input name="gender" id="female"type="radio">
+            Female</label>
+            <label for="Male">
+            <input name="gender" id="male"type="radio">
+            Male</label>
+            <label for="nonbinary">
+            <input name="gender" id="nonbinary" type="radio">
+            Nonbinary</label>
+            <label for="genderqueer/genderfluid">
+            <input name="gender" id="genderqueer/genderfluid"type="radio">
+            Genderqueer/Genderfluid</label>
+            <label for="prefer not to answer">
+            <input name="gender" id="prefer not to answer"type="radio">
+            Prefer not to answer</label>
+            <label for="self describe">
+            <input name="gender" id="self describe"type="radio">
+            Prefer to self describe</label>
+            <textarea name="" id="self describe"></textarea>
+  </fieldset>
+  
+  
+<fieldset>
+<legend>POWERS & ABILITIES</legend>
+<label for="powers">Describe your powers</label>
+<textarea name="" id="powers">Text here</textarea>
+<label for"rate""> Rate Your Abilities</label>
+<div class="three">
+<div class="strength">
+  <svg id="strength" viewBox="3.654 2.709 58.553 58.696" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This `svg` can't have the same `id` as the `select` below; `id`s must be unique.